packages/php/email-editor/src/Engine/Renderer/ContentRenderer/Postprocessors/class-border-style-postprocessor.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * This file is part of the WooCommerce Email Editor package
+ *
+ * @package Automattic\WooCommerce\EmailEditor
+ */
+
+declare(strict_types = 1);
+namespace Automattic\WooCommerce\EmailEditor\Engine\Renderer\ContentRenderer\Postprocessors;
+
+/**
+ * Ensures every border with border-width > 0 has a border-style, defaulting to 'solid' if missing, and removes extra border-style properties.
```

Comment:
Good job with this class. It is pretty complex. Please can you extend the class description about the purpose and add some examples?

Reply:
Added a more detailed description about the issue that this post processor resolves.